server/form-pages/apply/reasons-for-placement/basic-information/reasonForShortNotice.ts

```diff
@@ -35,7 +35,6 @@ export default class ReasonForShortNotice implements TasklistPage {
       this.title = 'Emergency application'
```

Comment:
Is it worth confirming with Dan that we definitely want a separate title and question for emergency applications but not short notice ones? The divergent behaviour for two types of non-standard application might be unexpected?

Reply:
Did Dan confirm that there needs to be a seperate title and question for emergency applications but not for short notice applications @vubalasu?

Reply:
yes i had a chat with him and confirmed